README.md

```diff
@@ -72,7 +73,7 @@ Le funzionalità elencate sono tutte intuitive e molto facili da utilizzare. La
  
 
 Inoltre ogni singolo messaggio viene salvato in un file *.txt* per soli scopi di debug in caso di malfunzionamenti del bot:
- - history_hub.txt
+ - log_AAAA_MM_GG.txt

```

Comment:
in una cartella?

Reply:
No no ... è un refuso delle versioni vecchie. Non ho modificato questa cosa a livello "backend" (viene già salvati nella cartella ./history_mozitabot/)